src/main/java/org/scalameter/annotations/benchmark.java

```diff
@@ -0,0 +1,31 @@
+package org.scalameter.annotations;
```

Comment:
Could you elaborate more on
>  and then expose them in the org.scalameter.api object, to make the available to Scala users.

? Should I just make type aliases in the `org.scalameter.api` for all annotations?